skynet/modules/ttt/summaries/v1/router.py

```diff
@@ -21,12 +24,16 @@ async def get_action_items(payload: DocumentPayload) -> JobId:
 
 @api_version(1)
 @router.post("/summary")
-async def get_summary(payload: DocumentPayload) -> JobId:
+async def get_summary(payload: DocumentPayload, request: Request) -> JobId:
     """
     Starts a job to summarize the given payload.
     """
 
-    return await create_job(job_type=JobType.SUMMARY, payload=payload)
+    bearer_jwt: JWTBearer = await JWTBearer().__call__(request)
```

Comment:
Where was auth happening up until now? Looks like the dependencies? Then aren't we auth-ing twice now?

Reply:
just decoding twice

Reply:
kind of, but more like decoding the jwt twice, since the pub key is cached anyway.

Reply:
Hum, perhaps we can rework the dependency as a decorator to avoid the double parsing in the future.